### PR TITLE
update readme with documentation 

### PR DESCRIPTION
**Pull request by sarah-altowaity1**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,80 @@
...
+
+### Import into a python module
+Use the following import statement at the very top of your .py file to use the functions from within your module. 
+```
+from pyfarmsay import pyfarmsay
+```
+or 
+
+```
+from pyfarmsay.pyfamrsay import [function name]
```
Comment (hpdouglass) [inline README.md:61]: Typo

File: README.md [hunk 2]
```diff
@@ -1,7 +1,80 @@
...
+1. cowsay (use 'cow' for the animal argument if you are invoking the function from the command line)
+2. pigsay (use 'pig' for the animal argument if you are invoking the function from the command line)
+3. chickensay (use 'chicken' for the animal argument if you are invoking the function from the command line)
+4. sheepsay (use 'sheep' for the animal argument if you are invoking the function from the command line)
+5. penguinsay (use 'penguin' for the animal argument if you are invoking the function from the command line)
+6. dogsay (use 'dog' for the animal argument if you are invoking the function from the command line)
+7. farmsay (use 'farm' for the animal argument if you are invoking the function from the command line)
+
+See our animals can only repeat what you tell
+them to say, so if no message is given, each animal will call out their native animal sound.
```
Comment (hpdouglass) [inline README.md:76]: I would make this one line, and just start it with "Out animals can..."

**Review by hpdouglass — COMMENTED** [review]
Two small typos but otherwise great!

File: README.md (hunk 1 above)
Comment (sarah-altowaity1) [reply README.md:61]: good catch - thanks!

**Pull request closed without merging**